Vic2ToHoI4/Source/Mappers/Graphics/GraphicsMapper.h

```diff
@@ -28,15 +28,15 @@ class GraphicsMapper
 	[[nodiscard]] std::vector<std::string> getMaleMonarchPortraits(const std::string& cultureGroup) const;
 	[[nodiscard]] std::vector<std::string> getFemaleMonarchPortraits(const std::string& cultureGroup) const;
 	[[nodiscard]] std::string getLeaderPortrait(const std::string& cultureGroup, const std::string& ideology);
+	[[nodiscard]] std::vector<std::string> getLeaderPortraits(const std::string& cultureGroup, const std::string& ideology) const;
```

Comment:
Add tests for these. See MapperTests/Graphics/GraphicsMapperTests.cpp for other examples.